src/performance/dashboard/src/modules/MetricsUtils.js

```diff
@@ -218,125 +208,92 @@ let getPathFromURL = function (url) {
  */
 let buildChartDataHTTP = function (params) {
 
-    const mcAPIData = params.projectMetrics;
     const scaleFactor = params.scaleFactor;
```

Comment:
nit: why not
```suggestion
    const { scaleFactor, counterName, urlFilter, decimals, filteredData } = params;
```
?

Reply:
> question: why not

that would work too,  likely they didn't always have the same name